README.md

```diff
@@ -118,6 +118,62 @@ export class AppComponent {
 }
 ```
 
+`arraySort` - sort an array by a specific property in individual elements
```

Comment:
Update the docs, please.

Reply:
Thanks for pointing it out, README is updated. :)

Yuchao